rules/xcodeproj.bzl

```
@@ -200,8 +200,7 @@ targets:
     install_script_sh = """set -eu
 readonly project_path="${PWD}/%s"
 readonly dest="${BUILD_WORKSPACE_DIRECTORY}/%s/"
-export TMPDIR="/tmp" # TODO: figure out why TMPDIR is not set
-readonly tmp_dest="${TMPDIR}/%s/"
+readonly tmp_dest=`mktemp -d`/%s/
```

```suggestion
readonly tmp_dest=$(mktemp -d)/%s/
```

in general it's better to use `$()` than backticks.